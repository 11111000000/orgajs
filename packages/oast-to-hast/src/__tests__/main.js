import u from 'unist-builder'
import toHAST from '..'

describe('Main', () => {
  it('POC', () => {
    const org = u('root', [
      u('section', { level: 1}, [
        u('heading', { level: 1, keyword: 'TODO', tags: ['shopping'] }, [
          u('text', 'remember the '),
          u('bold', [ u('text', 'milk') ])
        ]),
      ]),
      u('block', { name: 'SRC', params: ['javascript'] }, 'console.log("hello world")')
    ])
    expect(toHAST(org)).toMatchSnapshot()
  })

})
