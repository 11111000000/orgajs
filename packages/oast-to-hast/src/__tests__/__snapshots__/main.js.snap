// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main POC 1`] = `
Object {
  "children": Array [
    Object {
      "children": Array [
        Object {
          "type": "text",
          "value": "remember the milk",
        },
      ],
      "properties": Object {},
      "tagName": "h1",
      "type": "element",
    },
    Object {
      "children": Array [
        Object {
          "children": Array [
            Object {
              "type": "text",
              "value": "console.log(\\"hello world\\")",
            },
          ],
          "properties": Object {
            "className": Array [
              "language-javascript",
            ],
          },
          "tagName": "code",
          "type": "element",
        },
      ],
      "properties": Object {},
      "tagName": "pre",
      "type": "element",
    },
  ],
  "type": "root",
}
`;
